.github/workflows/pull_request_previews.yml

```diff
@@ -1,31 +1,33 @@
+# Synchronize pull requests to https://wptpr.live
+#
+# This task performs a potentially dangerous action (uploading code to
+# https://wptpr.live), so we restrict it to running on the base commit only
+# ('pull_request_target') so that malicious actors cannot modify the workflow
+# or scripts to force an upload.
 name: pr-preview-sync
```

Comment:
Nit: maybe rename it to `create-pr-preview` or something? "sync" kind of loses its original meaning with this change as it's no longer a cron job that "synchronizes" states.